### ISSUE TITLE
Threat indicator sounds

### ISSUE THREAD

**Issue by StevieGoodman** [CMP2804/Assessment2#17]
# Summary
In order to increase accessibility, threat indicators should have sound to alert players of danger.

## Subtasks
- [ ] Build up sound
- [ ] Alert sound

# Intended outcome
- A "build up" sound is played when an enemy character has the player character in their line-of-sight
- A "alert" sound is played when an enemy character fully detects the player character
